src/lib/commandBarConfigs/validators.ts

```diff
@@ -171,16 +172,26 @@ export const loftValidator = async ({
 }
 
 export const shellValidator = async ({
+  context,
   data,
 }: {
-  data: { selection: Selections }
+  context: CommandBarContext
+  data: { selection?: Selections; thickness?: KclExpression }
 }): Promise<boolean | string> => {
-  if (!isSelections(data.selection)) {
+  // Retrieve the thickness argument if it exists
+  const thicknessArgument = data.thickness
+    ? Number(data.thickness.valueCalculated)
+    : undefined
+  const thickness: Models['LengthUnit_type'] = thicknessArgument ?? 1e-9
+
+  // Retrieve the selection from the previous arg or this arg's data directly
+  const selection = data.selection ?? context.argumentsToSubmit['selection']
```

Comment:
Here we're making the validator capable for both the selection args and the thickness arg.

Reply:
Actually won't make it part of this PR, don't think this will be possible in a robust way pre rollback